rubygem-katello.spec

```diff
@@ -77,6 +77,7 @@ Requires: createrepo >= 0.9.9-18%{?dist}
 Requires: elasticsearch
 Requires: foreman >= 1.7.0
 Requires: java-openjdk >= 1:1.7.0
+Requires: java-openjdk < 1.8.0.45
```

Comment:
It works just fine with 1.8.0.31 which is what has been being pulled in for the last few months.